### PR TITLE
Fix Change Payment Method Button Link

### LINKED ISSUE #461: Change Payment Method Button [Android]
## Acceptance Criteria

- [ ] User should be able to select the Change Payment Method button on the Review Contributions screen.

## Context

Currently this button is difficult to select on Android.

### PR DESCRIPTION
**Pull request by richarddubay**
Fixes #461 

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [ ] Review new test logic

### CODE REVIEW

**Comment by richarddubay** [comment]
@conrad-vanl Can you please take a look at this and tell me what you think? I can rename the button link later, but I wanted to see if you thought this helped the issue. Seems like the button might be slightly more easy to tap now.

File: src/@ui/forms/PaymentConfirmationForm.js [hunk 1]
```diff
@@ -54,6 +54,30 @@ const ButtonLinkContainer = styled(({ theme }) => ({
   justifyContent: 'center',
 }))(View);
 
+const BigButtonLink = compose(
+  withProps({
+    zIndex: 100,
+    style:
+      Platform.OS === 'android'
+        ? {
+          paddingRight: 10,
```
Comment (IsaacHardy) [inline src/@ui/forms/PaymentConfirmationForm.js]: `padding: 10` would work, too.

Comment (richarddubay) [reply src/@ui/forms/PaymentConfirmationForm.js]: Haha...you right. 😆

File: src/@ui/forms/PaymentConfirmationForm.js [hunk 2]
```diff
@@ -54,6 +54,30 @@ const ButtonLinkContainer = styled(({ theme }) => ({
   justifyContent: 'center',
 }))(View);
 
+const BigButtonLink = compose(
+  withProps({
+    zIndex: 100,
+    style:
```
Comment (burkeshartsis) [inline src/@ui/forms/PaymentConfirmationForm.js]: what if we did this as a styled component instead 😎

**Comment by richarddubay** [comment]
Web before changes:
screenshot 2018-04-12 14 33 07

Web after changes:
screenshot 2018-04-12 14 34 37

iOS before changes:
screenshot 2018-04-12 14 29 01

iOS after changes:
screenshot 2018-04-12 14 39 54

Android before changes:
screenshot 2018-04-12 14 25 16

Android after changes:
screenshot 2018-04-12 14 41 49

**Comment by richarddubay** [comment]
@burkeshartsis This is ready for review again.

**Pull request merged**